stdlib/stdlib.mli

```diff
@@ -19,9 +19,10 @@
     compilation. All components of this module can therefore be
     referred by their short name, without prefixing them by [Stdlib].
 
-    It particular, it provides the basic operations over the built-in types
-    (numbers, booleans, byte sequences, strings, exceptions, references,
-    lists, arrays, input-output channels, ...).
+    It particular, it provides the basic operations over the built-in
+    types (numbers, booleans, byte sequences, strings, exceptions,
+    references, lists, arrays, input-output channels, ...) and the
+    {{!modules}standard library modules}.
```

Comment:
I think this should be OK, because the syntax for module listings is `{!modules:...}`, but it would be nice if you could confirm that this does indeed work.

Reply:
Built the htmlman locally and yes it does work.